app/models/author.rb

```diff
@@ -1,6 +1,8 @@
 class Author < ApplicationRecord
-  has_many :books
-  has_many :published, foreign_key: :publisher_id, class_name: 'Book', as: :publisher
+  has_many :books, dependent: :destroy
```

Comment:
I have added the dependent destroy here to make sure when i delete an author i delete his books as well to keep my data consistent

Reply:
Not really i usually always do it this way, i would love to hear more if there is a better way 👍

Reply:
Done 👍